src/pantry.js

```diff
@@ -4,4 +4,4 @@ class Pantry {
   }
 }
 
-export default Pantry;
+module.exports = Pantry;
```

Comment:
How come you decided to change this? I think we need `export default` syntax for Webpack to work correctly, but I could be thinking of something different!

Reply:
I think you're right, but I was having some issues getting it to connect with the 'export.' Let me re-visit it.